coding-guides/sqlite-based-points-system.md

```diff
@@ -23,17 +23,17 @@ For this example points system we want the user's ID, points and level to be com
 Our starting point is a very basic message handler with pre-existing commands - such as what we see in the [Command with Arguments](../first-bot/command-with-arguments.md) page of this guide. The code is as such:
 
 ```javascript
-const Discord = require("discord.js");
-const client = new Discord.Client({
-  intents: ["GUILDS", "GUILD_MESSAGES"]
+const { Client, Intents: { FLAGS: { GUILDS, GUILD_MESSAGES } } } = require("discord.js");
+const client = new Client({
+  intents: [GUILDS, GUILD_MESSAGES]
 });
```

Comment:
To keep things consistent with this line here: [`sqlite-based-points-system.md`](https://github.com/AnIdiotsGuide/discordjs-bot-guide/pull/170/files#diff-51c8636766f0af606d940c35b829646f47506cc2009cfd132821dc07ed254a67R69)

```suggestion
const { Client, Intents } = require("discord.js");

const client = new Client({
  intents: [Intents.FLAGS.GUILDS, Intents.FLAGS.GUILD_MESSAGES]
});
```

Reply:
Done :+1: